src/data/validation/email.test.js

```diff
@@ -1,16 +1,25 @@
 import _ from 'lodash';
 import {
   extractSalesforceIds,
+  returnValidatedEmails,
   validateEmailAddresses,
   validateEmailAddressesFields,
+  validateEmailAddrTemplateForm,
   validateEmailTemplateFields,
   validateEmailTemplateForm,
-  validateEmailAddrTemplateForm,
-  returnValidatedEmails,
 } from './email';
-import { EMAIL_ADDRESS_TEXT_FORM_DATA, EMAIL_ADDRESS_CSV_FORM_DATA } from '../constants/addUsers';
+import { EMAIL_ADDRESS_CSV_FORM_DATA, EMAIL_ADDRESS_TEXT_FORM_DATA } from '../constants/addUsers';
 import { EMAIL_TEMPLATE_SUBJECT_KEY, OFFER_ASSIGNMENT_EMAIL_SUBJECT_LIMIT } from '../constants/emailTemplate';
 
+const hasFormatChar = (str) => [...str].some(char => {
```

Comment:
This function was added to prevent "self fulfilling" test cases where using the same regular expression to validate the regular expression is not actually testing the regular expression